src/networking.c

```diff
@@ -3876,7 +3881,37 @@ void unblockPostponedClients() {
     }
 }
 
-/* Pause clients up to the specified unixtime (in ms) for a given type of
+/* As pause-client feature refactored internally to become pause-actions,
+ * this is a wrapper that keeps pause-client backward compatible logic:
```

Comment:
i don't think the code should document that it got refactored (that's for the commit comment).
i think it is enough to state what this function does (i.e. update a client pause by using a "max" on the existing pause and the new pause, as you describe below.